### DISCUSSION TITLE
Can I use a formular field with javascript to query other (bbdb) tables?

### DISCUSSION THREAD

**Discussion by gerhardmpl** [Budibase/budibase#4474] [Help] [answered]
Is it possible to add a formular column to one table, that reads a field value from the current row and queries another (bbdb) table or view with SQL and Javascript? I would like to add some additional cross table logic.

**Comment by mjashanks** [comment, marked as answer]
Not at the table level. You can do this in the `Design` section, by combining `Data Providers` together.
